cg/meta/workflow/nf_analysis.py

```diff
@@ -803,3 +805,34 @@ def get_cases_to_store(self) -> list[Case]:
             if self.trailblazer_api.is_latest_analysis_completed(case_id=case.internal_id)
             or self.trailblazer_api.is_latest_analysis_qc(case_id=case.internal_id)
         ]
+
+    def clean_workflow_run_dir(self, yes: bool, case_id: str, dry_run: bool = False):
```

Comment:
this method is almost the same as `clean_run_dir` can we move it to analysis_api and refactor to avoid duplicating methods?